### PR TITLE
Docs.installation

### PR DESCRIPTION
**Pull request by pandersity**
Update installation docs to all point to the same place. Plus a few formatting/reorg suggestions to make docs more readable.

### CODE REVIEW

**Comment by dhdaines** [comment]
This seems helpful, but it also seems to imply that `poetry` is *required* for development which is assuredly not the case.  I found `poetry` while convenient to be resource-hungry and full of bugs when I tried to use it, but perhaps it has improved since then?

**Comment by dhdaines** [comment]
Oops!  I was thinking of the old `roedoejet/mothertongues` repository ... now I see that Poetry is now required here!  Ignore my last comment...

File: docs/install.md [hunk 1]
```diff
@@ -16,30 +20,4 @@ pip install mothertongues
...
-# Clone repo and UI submodule
-git clone https://github.com/MotherTongues/mothertongues.git --recursive
-# Build the UI:
-cd mothertongues/mothertongues-UI && npm install
-# Build the Python Development version of the UI:
-npx nx build mtd-mobile-ui --configuration=pydev
-# Install the Python package:
-cd .. && poetry install
-```
+    If you are developer looking to install locally, see this [Contributing page](/Contributing.md)
```
Comment (roedoejet) [inline docs/install.md:23]:
```suggestion
    If you are an advanced developer looking to install locally, see this [Contributing page](/Contributing.md)
```

**Review by roedoejet — APPROVED** [review]
Looks good to me. Thanks @pandersity !

**Pull request merged**